assets/js/core/connection.js

```diff
@@ -87,7 +87,8 @@
 
     // It is currently not possible to specify "name"
     var attrs = {
-      url: this.url
+      url: this.url,
+      on_connect_commands: this.on_connect_commands,
```

Comment:
I think `on_connect_commands` should be defined inside the constructor as well, so we can watch it from Vue.